stackit/services/mariadb/instance/resource.go

```diff
@@ -292,6 +292,10 @@ func (r *instanceResource) Read(ctx context.Context, req resource.ReadRequest, r
 		core.LogAndAddError(ctx, &resp.Diagnostics, "Error mapping fields", err.Error())
 		return
 	}
+
+	// Compute and store values not present in the API response
+	r.loadPlanNameAndVersion(ctx, &diags, &state)
```

Comment:
We should check for errors after `loadPlanNameAndVersion`.
The question is that if the `Read` should fail when loading the name and version fails. I would say it shouldnt, but maybe a warning would be good. In that case, I would make line 635 below a warning instead of error

Reply:
Agree, added a check and early return in case the call to `GetOfferings` fails but kept the warning if for some reason the `plan_name` and `version` can't be found for the returned `plan_id`. This way it doesn't block the normal functioning of the TFP (as it's only relevant for the import) and there the user can then recover by getting the fields from the Portal e.g.